mteb/evaluation/MTEB.py

```diff
@@ -41,20 +41,14 @@ def __init__(
         of tasks to be evaluated with the `tasks` argument. If
         `tasks` is specified, we filter tasks based on `task_langs`
 
-        Parameters
-        ----------
-        task_types: list of str / None
-            List of task types (Clustering, Retrieval..) to be evaluated. If None, all tasks will be evaluated
-        task_categories: list of str / None
-            List of task categories (s2s, p2p..) to be evaluated. If None, all tasks will be evaluated
-        task_langs: list of str / None
-            List of languages to be evaluated. if None, all languages will be evaluated. ["en", "de"] will evaluate on "en", "de", "en-de" and "de-en"
-        tasks: list of str / None
-            List of tasks to be evaluated. If specified, we filter tasks based on `task_langs` only
-        version: int / None
-            Version of the benchmark to use. If None, latest is used
-        err_logs_path: str / None
-            Path to save error logs
+        Args:
+            task_types: List of task types (Clustering, Retrieval..) to be evaluated. If None, all tasks will be evaluated
```

Comment:
Since this takes a list of [str], we might want to specify where to find valid inputs.

Reply:
Ah, I see you only did formatting changes. Feel free to disregard if this is acceptable.